jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiRequest.java

```diff
@@ -1049,7 +1035,7 @@ public BufferedReader getReader() throws IOException
 
         String encoding = getCharacterEncoding();
         if (encoding == null)
-            encoding = StringUtil.__ISO_8859_1;
+            encoding = MimeTypes.ISO_8859_1;
```

Comment:
Why is a Charset constant on the `MimeTypes` class?  Seems like a weird place for that.
Charset deals with encodings of bytes to Strings (and back), why not have these Charset constants on `StringUtil`?

Reply:
@joakime Because the charset has nothing to do with strings in general, and everything to do with mime types that often have a charset argument: e.g. `text/html;charset=utf-8`
If I was going to be more radical, I'd rename `MimeTypes` to `ContentTypes` as strictly speaking a content type is a mimetype plus an optional charset.   But this PR is about util, not http.

Reply:
That's only true for mime types that have a character representation.

Take the mime-type `application/octet-stream`, that has no charset, and never will.
Nor will `image/png` or `image/jpeg` and similar.

Reply:
Yeah but charset has a lot more to do with mymetypes than it does general strings